src/apps/containers/components/EditMenu.tsx

```diff
@@ -30,6 +39,13 @@ export const EditMenu: React.FC<Props> = ({backUrl}: Props) => {
                         Discard changes and view
                     </Link>
                 </MenuItem>
+                {hasDelete && 
+                <MenuItem>
+                    <Button onClick={openModal} className={classes.deleteButton}>
+                        Delete Organisation
```

Comment:
The text "Delete Organisation" shouldn't be embedded in a component outside of the organisation app. Similarly, the way this renders in my browser seems off:

<img width="224" alt="Screenshot 2021-01-11 at 15 55 43" src="https://user-images.githubusercontent.com/52504170/104237089-72cb2e80-544f-11eb-8129-8460178e0868.png">

I.e., the text is smaller and there's some additional padding. I would think that we should make this look more like the "Retire Concept" action when editing a concept. I also don't think we need to make the text red, but that's just my opinion.

Reply:
Agreed with Ian's comments; this text should be the same size, style, and color as the "Discard changes and view" option. We can sleep in peace though because we know that if a user accidentally clicks "Delete Organization", they will see the confirm modal. Right?